src/ui/application-window.vala

```diff
@@ -330,12 +332,30 @@ namespace Peek.Ui {
         if (get_window_width () < SMALL_WINDOW_SIZE) {
           GtkHelper.hide_button_label (record_button);
           GtkHelper.hide_button_label (stop_button);
+          //Set the scale of shortcut_label
+          Pango.AttrList attrs = new Pango.AttrList (); 
+          attrs.insert (Pango.attr_scale_new (Pango.Scale.SMALL)); 
+          shortcut_label.attributes = attrs;
         } else {
           GtkHelper.show_button_label (record_button);
           GtkHelper.show_button_label (stop_button);
+          //Set the scale of shortcut_label
+          Pango.AttrList attrs = new Pango.AttrList (); 
+          attrs.insert (Pango.attr_scale_new (Pango.Scale.LARGE)); 
+          shortcut_label.attributes = attrs;
         }
 
         if (!is_recording) {
+          var shortcut =Application.get_app_settings();
+          string keys =shortcut.get_string("keybinding-toggle-recording");
```

Comment:
You should be able to replace the custom handling of the shortcut string by using Gtk's builtin functions for that:

```
string keys = shortcut.get_string ("keybinding-toggle-recording");
uint accelerator_key;
Gdk.ModifierType accelerator_mods;
Gtk.accelerator_parse (keys, out accelerator_key, out accelerator_mods);
var shortcut_hint = Gtk.accelerator_get_label (accelerator_key, accelerator_mods);
shortcut_label.set_text ("Start/Stop: " + shortcut_hint);
```

See also shortcut-label.vala where I also used this to display the shortcut on older Gtk versions.

Reply:
well that makes it easier thanks!